src/metabase/api/dashboard.clj

```diff
@@ -623,7 +702,8 @@
                                    :present #{:description :position :width :collection_id :collection_position :cache_ttl}
                                    :non-nil #{:name :parameters :caveats :points_of_interest :show_in_getting_started :enable_embedding
                                               :embedding_params :archived :auto_apply_filters}))]
-              (t2/update! Dashboard id updates))
+              (t2/update! Dashboard id updates)
+              (handle-broken-subscriptions id original-params))
```

Comment:
Should we conditionally `handle-broken-subscriptions` only when `:parameters` change? Or do we effectively get a no-op if no parameters change already?

Reply:
Done.